caput/mpiarray.py

```diff
@@ -712,12 +770,11 @@ def to_hdf5(
 
         Parameters
         ----------
-        filename : str, h5py.File or h5py.Group
+        f : str, h5py.File or h5py.Group
             File to write dataset into.
         dataset : string
             Name of dataset to write into. Should not exist.
```

Comment:
Needs documentation for all parameters.

Reply:
Oh *wow*, they are not documented anywhere.

Reply:
Still need doc entry for `chunks`